frontend/src/pages/UpdateUser.tsx

```diff
@@ -1,5 +1,119 @@
-import { Box } from '@mui/material';
+import {
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  Button,
+  FormControl,
+} from '@mui/material';
+import Background from '../components/Background';
+import { useState } from 'react';
 
 export const UpdateUser = () => {
-  return <Box>This is the UpdateUser Page!</Box>;
+  const [userDetails, setUserDetails] = useState<{
+    emojis: string;
+    faculty: string;
+    bestEat: string;
+    interests: string;
+  }>({ emojis: '', faculty: '', bestEat: '', interests: '' });
+
+  const handleChange = (e: any) => {
+    const { name, value } = e.target;
+    setUserDetails({
+      ...userDetails,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    console.log(userDetails);
+    // TODO:
+    // Hit the correct API to successfully store info to the db
+    const res = await fetch('http://dynamodb:8000/updateUser', {
+      method: 'POST',
+    });
+    const data = await res.json();
+    console.log(data);
+  };
+
+  const baseFlexStyles = {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  };
+
+  return (
+    <Background bgcolor={'#fff'}>
+      <Grid item xs={12} sx={baseFlexStyles}>
```

Comment:
In general, we don't want a situation where we only have 1 child grid item for a parent. Like this:
--Parent Grid container
----Child Grid item

If it's like this, then we don't even need the child. Currently we've got it like above, the background is the container and the updateuser page is 1 big grid item. We should have something closer to this:
--Parent Grid container (Background)
----Child Grid item 1 (the heading)
----Child Grid item 2 (the form)